Changelog.md

```diff
@@ -1,6 +1,7 @@
 # Next
 ### Added
 - **Breaking Change** `.parameterEncoding(Swift.Error)` case to `MoyaError`. [#1248](https://github.com/Moya/Moya/pull/1248) by [@SD10](https://github.com/SD10).
+- Added `Decodable` object mappings. #1335 by [@devxoul](https://github.com/devxoul).
```

Comment:
Added `Decodable` object mapping methods to `Moya.Response`. 
This could be a little more descriptive. Sorry to be so needy 😅

Reply:
That makes sense!